components/style/properties.mako.rs

```diff
@@ -6498,23 +6457,17 @@ fn cascade_with_cached_declarations(
                                             DeclaredValue::WithVariables { .. } => unreachable!()
                                         }, &mut error_reporter
                                     );
-                                    Arc::make_mut(&mut style_${style_struct.ident})
+                                    Arc::make_mut(&mut context.style.${style_struct.ident})
                                         .${property.ident} = computed_value;
                                 % endif
 
                                 % if property.name in DERIVED_LONGHANDS:
-                                    % if not style_struct.inherited:
```

Comment:
Yes. In this function non-inherited styles are cached so we don’t need to run the cascade for them again. `let computed_value = …;` is in a larger block above is in `% if style_struct.inherited:` (*without* `not`).

Before this PR, `computed_value` was needed unconditionally, so this `% if not style_struct.inherited:` block (*with* `not`) defined it for the rest of the cases. Now that `computed_value` is not used in the `derive_from_*` call anymore, this second `let computed_value = …;` statement would be dead code.